pkg/packagekit/updates.jsx

```diff
@@ -771,24 +773,22 @@ const StatusCard = ({ updates, highestSeverity, timeSinceRefresh, tracerPackages
         notifications.push({
             id: "processes-need-restart",
             stateStr: _("Some software needs to be restarted manually"),
-            icon: <ProcessAutomationIcon id="icon" size="md" />,
+            icon: <ProcessAutomationIcon />,
             secondary: <Text component={TextVariants.small}>{tracerPackages.manual.join(", ")}</Text>
         });
     }
 
     return (<>
         { notifications.map(notification => (
-            <Flex direction={{ default: 'column', sm: 'row' }} key={notification.id} id={notification.id} className="status-notification">
-                <FlexItem>{notification.icon}</FlexItem>
-                <Flex flex={{ default: 'flex_1' }} direction={{ default: 'column' }}>
-                    <FlexItem>
-                        <Text id="state" component={TextVariants.p}>{notification.stateStr}</Text>
-                    </FlexItem>
-                    <FlexItem>
+            <Split key={notification.id} id={notification.id} hasGutter>
```

Comment:
This is breaking sometimes:
![Screen Shot 2021-03-29 at 16 32 30](https://user-images.githubusercontent.com/14921356/112853192-e541c700-90ac-11eb-9c3d-d40c37598f97.png)

Try:
`<Flex flexWrap={{ default: 'nowrap' }}>`
Seems to work.

Reply:
How does it break?

Reply:
Ah sorry that I did not answer, missed the notification. I just meant the alignment of the icon with the text on the right, I guess you fixed it already according to the screenshots.